cron.go

```diff
@@ -180,6 +180,9 @@ func (c *Cron) run() {
 
 // Stop the cron scheduler.
```

Comment:
I suggest documenting the behavior of Stop if it's not already running. Something like this:

```
// Stop the cron scheduler, if it's running. Otherwise this does nothing.
```

People should know what happen if they call Stop just by reading the docs, without having to look at the code (the implementation details). IMO it's worthwhile to document how Stop behaves if it's not running, whether it be a fatal error or a harmless no-op (which it it will be now).